cves/CVE-2014-0050.yml

```diff
@@ -43,7 +43,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: A software package offered by Apache Tomcat for hosting websites 
+  allowed attackers to consume near 100% of the host's CPU, creating a Denial of
+  Service attack. This exploit involved an attacker tinkering with the 
+  Content-Type header when uploading a file to the server, preventing the system
+  from exiting an infinite loop.
```

Comment:
It might be helpful to add a description of what exactly triggers the exploit.

Reply:
I'll add something about that: it was mostly due to changing the boundaries in the request.